small QOL changes to kilo's genetics, xenobio, and botany

Dabger1 opened the pull request:
## About The Pull Request

adds two additional slime pens and expands the kill chamber by one column (pushed into maints a tad) and removed the maint door to the kill room, and moving the disposals chute so its not in the way in kilo's xenobio, along with increasing space in botany by removing the south wall to the seed vendors and lockers, along with expanding botany maints by removing the glass and sand area just below botany to make space for plumbing machines and the like, and adds one extra DNA scanner to genetics by pushing back the monkey pen by one tile.

## Why It's Good For The Game

kilo sucks to play on, especially for botany, genetics and xenobio since its so cramped, this aims to make it less cramped for those that play in those areas without changing the layout of the station

## Pics from map editor of changes
![genetics new kilo](https://user-images.githubusercontent.com/31255692/199883156-65c1d0ff-d5a9-453c-8c1e-ffcd8c6cf2cd.PNG)
![botany new kilo](https://user-images.githubusercontent.com/31255692/199883160-2b7e4863-d28e-48d0-ba88-8a0c8cd35288.PNG)
![xenobio new kilo](https://user-images.githubusercontent.com/31255692/199885442-ffa62e63-6d33-4c10-8a3d-1674bf7d1ddb.PNG)



:cl:
add: added Two xenobio pens to kilo station
add: added an extra DNA scanner to genetics on kilo station
qol: made botany and its maints less cramped on kilo station and moved the disposal chute in kilo's xenobio
/:cl:

Dabger1 committed:
small QOL changes to kilo's genetics, xenobio, and botany

Dabger1 commented:
forgot to update the botany pic, the version i uploaded doesn't have the firelock, floating cam, and floating button

san7890 commented:
In your CL, you'll want to add what you changed such that it shows up to players. Like:

:cl:
add: Adds two more slime pens to KiloStation's Xenobiology Department
fix: ...
/:cl:

You can repeat add/fix/qol/del as much as you need.

MrMelbert reviewed (commented):
![image](https://user-images.githubusercontent.com/51863163/199904248-b3db0695-1169-4529-8baf-bb09e3cc4682.png)

Looks like some things got left behind:
- Stripe decal in wall
- Light fixture in wall
- Poster in wall (maybe?)
- Disposals / cable / oxygen piping doesn't connect

MrMelbert reviewed (commented):
![image](https://user-images.githubusercontent.com/51863163/199904435-9a2717b2-a92d-4fab-a345-5b8fb9394856.png)

Third genetics console could use a seat, also missing purple siding like the other two

MrMelbert reviewed (commented):
![image](https://user-images.githubusercontent.com/51863163/199904591-14b8f23f-dcd6-421e-90e2-02a9c8403cd3.png)

Missed a turf here, on the right

![image](https://user-images.githubusercontent.com/51863163/199904660-8cd5a81f-d3eb-4e6f-8c0c-eee99ba947cc.png)

This area looks very plain now and could use some re-adjustments.

MrMelbert reviewed (commented):
![image](https://user-images.githubusercontent.com/51863163/199904815-83621ba1-2cbf-45f6-8c7c-677eb71779b2.png)

Consistency pass for the floor decalling here. Keep the pattern alive!

Also consider adding a cool floor design in the open space.

MrMelbert reviewed (changes requested):
Cool start!

MTandi commented:
This area under the tables indeed looks weird now, maybe you should bring back that nice 2 tile garden here. It was a nice touch.

And why did you remove the sand bit and left an empty maint area there? It was cool, and you didn't really need this for hydroponics expansion.

You barely changed the hydroponics. It looks more ugly now and lost a wall to the closet. But maybe you wanted to expand the whole area downward to use the space from removed windows? And I'm personally against removal of these. It's a cool touch to the map that reminds that you are on an asteroid.

![image](https://user-images.githubusercontent.com/3625094/199921715-7cc9c3fa-3e39-487f-8286-b71af3637d4f.png)

Please consider swapping tables and the workstation in genetics. The person using the console should not be in a way to the pens.

![image](https://user-images.githubusercontent.com/3625094/199923169-65115216-a4a9-4568-9168-cec456c5ca27.png)

I like the fact that xenobio will have a bit more space, but now the area looks a bit unbalanced - it's packed with stuff on the bottom, and empty on the top. 

I think it would be nicer if some consoles/grinders moved to top, to balance the space.

Kinda like this, but you need to think through the whole area and the workplaces to not make people run around a lot.

![image](https://user-images.githubusercontent.com/3625094/199924028-28a176e9-7d50-40b3-b590-286d2e4e8f8a.png)

Thank you for the contribution, but I think that you need to take into account the general style of the map, the ergonomics and maybe refer to how these areas are done on other maps.

Livrah commented:
Xenobio really needs glass walls between the pens.

Dabger1 commented:
update to the changes made (before adding to the PR): changed the floors in sci to normal tile instead of showroom, rotated the new DNA console so its not in the way, moved a slime console up into the new area, along with a new slime processor, moving the biohazard closets into the new area, moved the monkey recycler, and added some tile art of a fire extinguisher.
re-added the grassy patch in botany.

the main reason i wanted to expand botany was to help with the crowding issue that arises when an automated setup for basic botany chems is added.

i've been talking with the mappers in the discord about the upload topic, would be easier to chat there about it.



![genetics new kilo](https://user-images.githubusercontent.com/31255692/200050157-7b465e75-d59a-4cf5-9eb4-21cca7af70b2.PNG)
![sci floors](https://user-images.githubusercontent.com/31255692/200050158-b6c7bb36-3e34-4d6c-96aa-8893380293f1.PNG)
![xenobio new kilo](https://user-images.githubusercontent.com/31255692/200050159-a6e2a684-eeaf-4253-9eef-e7a6901f1b05.PNG)

Dabger1 commented:
thank you all for your feedback and criticism! due to some extra changes i made, i'll close this PR and open a new one containing updated info and an updated map file. again thank you for sparing some time to give feedback, as this was my first go at map editing and my first PR.
